src/richie/apps/courses/templatetags/extra_tags.py

```diff
@@ -235,6 +235,7 @@ def course_enrollment_widget_props(context):
                 "id": course_run.id,
                 "resource_link": course_run.resource_link,
                 "priority": course_run.state["priority"],
+                "start": course_run.start.isoformat(),
```

Comment:
@sampaccoud @jbpenrath what do you think about replacing `start` attribute with `has_started` or `starts_in` : 
both permits to replace frontend test `new Date().toISOString() >= courseRun.start` and prevent timezone issues.
`has_started` would be a boolean.
`starts_in` would be remaining days before the course start, and we could display this value in front

Reply:
Smart and seems legit. I'll let @jbpenrath comment on the possibility to compare this date to now in the frontend (since the isoformat returned by the backend is timezone aware, it should be feasible).

Reply:
In fact, I don't see any reason to compare user local time with the course start datetime

Reply:
If the local time is timezone aware you can compare them to know if the course is open at a given time. But I agree that it is easier on the backend...

Reply:
You are right @kernicPanel, as we only use start date to check if course started, backend could be in charge to send this information. I think it is a good idea to send `starts_in` and add a message below `You are enrolled in this course run`: `The course will start in X days`.

Reply:
ok, great ! Thanks for pointing that lib @jbpenrath